src/opm/output/eclipse/Summary.cpp

```diff
@@ -57,76 +57,70 @@
 #include <iterator>
 #include <memory>
 #include <numeric>
+#include <regex>
 #include <stdexcept>
 #include <string>
 #include <unordered_map>
 #include <utility>
 #include <vector>
 
 namespace {
-    struct ParamCTorArgs
-    {
-        std::string kw;
-        Opm::SummaryConfigNode::Type type;
-    };
-
-    std::vector<ParamCTorArgs> requiredRestartVectors()
+    std::vector<std::pair<std::string, Opm::EclIO::SummaryNode::Type>> requiredRestartVectors()
```

Comment:
No.  We're not using `std::pair` for this.  It is infinitely better to have a structure with proper names than to refer to `.first` and `.second`.

Reply:
Reverted.